README.md

```diff
@@ -1,43 +1,87 @@
-# bot2
+# Bot2
 
+A discord bot made with [interactions.py](https://github.com/interactions-py/interactions.py).
+Visit [the official website](https://interactions-py.github.io/interactions.py/) to get started.
 
+## Development installation
 
+<details>
+<summary>Click to expand</summary>
 
-A discord bot made with [interactions.py](https://github.com/interactions-py/interactions.py).
-Visit [the official website](https://interactions-py.github.io/interactions.py/) to get started.
+### Mongodb
```

Comment:
capitalize 'DB'

Reply:
oops, I missed these comments before merging, somehow. I'll catch this capitalization on the next PR.